src/ForDev.tsx

```
@@ -425,29 +428,30 @@ export default function ForDev() {
                         const style = {
-                            width: `${(state.state / 1023) * 100}%`,
+                            width: `${(state.switchState / 1023) * 100}%`,
                         };
 
                         return (
                             <div
                                 className={`${className} relative mb-1 flex place-content-between items-center rounded-md border-2 px-4 transition-colors`}
                             >
                                 <div className="z-10">
-                                    {`${pin.toString().padStart(2, "0")} : ${state.state}`}
+                                    {`${pin.toString().padStart(2, "0")} : ${state.switchState}`}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)